todo_app/app.py

```diff
@@ -1,11 +1,23 @@
-from flask import Flask
+from flask import Flask, request, redirect, render_template
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    #print(items)
+    return render_template('index.html', items=items)
+    
+@app.route('/additem', methods=['POST'])
+def addnewtodo():
+    newtodo= request.form.get('nitem')
+    if newtodo != 'Enter ToDo':
+        add_item(newtodo)
```

Comment:
To have placeholder text that users can't turn into a real item, use `placeholder="Enter ToDo"` instead of `value="..."` in your index.html